manifest_staging/charts/aad-pod-identity/values.yaml

```diff
@@ -69,8 +69,18 @@ mic:
     kubernetes.io/os: linux
 
   tolerations: []
+    # - key: "CriticalAddonsOnly"
+    #   operator: "Exists"
 
   affinity: {}
+    # nodeAffinity:
+    #   preferredDuringSchedulingIgnoredDuringExecution:
+    #     - preference:
+    #       matchExpressions:
+    #         - key: kubernetes.azure.com/mode
+    #           operator: In
+    #           values:
+    #             - system
```

Comment:
Sorry, should've posted earlier. When trying to apply the above I found I needed a `weight` key - e.g.
```yaml
  priorityClassName: "system-cluster-critical"
  tolerations:
    - key: "CriticalAddonsOnly"
      operator: "Exists"
  affinity:
    nodeAffinity:
      preferredDuringSchedulingIgnoredDuringExecution:
        - weight: 1
          preference:
            matchExpressions:
            - key: kubernetes.azure.com/mode
              operator: In
              values:
                - system
```
Some refs that helped me figure it out:
* https://docs.openshift.com/container-platform/3.6/admin_guide/scheduling/node_affinity.html
* https://kubernetes.io/docs/concepts/scheduling-eviction/assign-pod-node/#node-affinity

Reply:
Ah I missed that. Would you like to open a PR for that

Reply:
Done in #996!